website/src/main/tut/docs/installation/sbt.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: docs
+title: sbt-scalafix
+---
+
+## sbt-scalafix
+The sbt-plugin is the recommended integration for semantic rules.
+
+```scala
+// ===> project/plugins.sbt
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "{{ site.stableVersion }}")
+
+// ===> build.sbt
+scalaVersion := "{{ site.scala212 }}" // {{ site.scala211 }} is also supported.
+// if scalacOptions is defined with append `++=`, do nothing.
+// if scalacOptions is defined like this: scalacOptions := List(...),
+// then do one of the following
+scalacOptions ++= List(...) // change := to ++=
+// or
+scalacOptions := List(...)                    // keep unchanged
+scalacOptions ++= scalafixScalacOptions.value // add this line
+
+// ===> sbt shell (example usage)
+> scalafix                               // Run .scalafix.conf rules
+> scalafix RemoveUnusedImports           // Run specific rule
+> myProject/scalafix RemoveUnusedImports // Run rule in one project only
+> test:scalafix RemoveUnusedImports      // Run rule in single configuration
+> scalafix ExplicitR<TAB>                // use tab completion
+> scalafix replace:com.foobar/com.buzbaz // refactor (experimental)
+> scalafix file:rules/MyRule.scala       // run local custom rule
+> scalafix github:org/repo/v1            // run library migration rule
+```
+
+### Verify sbt installation
+To verify the installation, check that the scalacOptions include -Xplugin-require:semanticdb
+
+```scala
+> show scalacOptions
+[info] * -Yrangepos                  // required
+[info] * -Xplugin-require:semanticdb // recommended
+[info] * -P:semanticdb:sourceroot:/x  // recommended
+> show libraryDependencies
+[info] * org.scalameta:semanticdb-scalac:{{ site.scalametaVersion }}:plugin->default(compile)
+```
+
+### scalafix-sbt-example
+For a minimal working example usage of sbt-scalafix, see the [scalacenter/scalafix-sbt-example](https://github.com/scalacenter/scalafix-sbt-example) repository.
+
+```scala
+git clone https://github.com/olafurpg/scalafix-sbt-example
+cd scalafix-sbt-example
+sbt "scalafix RemoveUnusedImports"
+git diff // should produce a diff
+```
+
+### sbt settings and tasks
+
+| Name | Type | Description
+|------|------|-------------
+| `scalafix <rule>..` | `Unit` | Run scalafix on project sources. See {% doc_ref Rules %} or use tab completion to explore supported rules.
```

Comment:
I've considered including these keys from scalafix-sbt/src/main/scala/.../ScalafixPlugin.scala's `autoImport`, to keep it in sync with the actual plugin. Nothing to change now, just an idea for the future. If the sbt plugin doesn't change much it may not be worth it.